### PR TITLE
Sort by date

### PR DESCRIPTION
**Pull request by rajat19**
# Related Issue
https://github.com/frankcash/HackerQueue/issues/63
Posts are currently sorted by how they are crawled from respective sites.
Need to be sorted by the time of them being posted in their respective sites

# Changes Proposed
- Added published at column in table
- Received and Stored time from respective sites
- Sort by published_at in main.jade

### CODE REVIEW

**Comment by frankcash** [comment]
There appears to be conflicts with master, can you fix them before I resolve?

Also, would this conflict with #64

**Comment by rajat19** [comment]
Ok I will resolve then

**Comment by rajat19** [comment]
I would recommend that you first resolve and merge https://github.com/frankcash/HackerQueue/pull/64
Then I would resolve my conflicts after testing if any problem emerges

**Comment by rajat19** [comment]
All checks have passed
And without any conflict with master
@frankcash you can now check my changes and approve this PR

File: routes/lobsters.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ var parseLobsterElement = function(a) {
   }
 
   const QUERY = 'INSERT INTO "crawls" ("story_url", "source", "title", "comments", "crawled_at", "published_at") VALUES($1, $2, $3, $4, $5, $6) ON CONFLICT DO NOTHING;';
-  db.query(QUERY,[url, source, title, comments_link, new Date() ], function (err, res) {
+  db.query(QUERY,[url, source, title, comments_link, new Date() ], function (err) {
```
Comment (frankcash) [inline routes/lobsters.js]:
```sql
INSERT INTO "crawls" ("story_url", "source", "title", "comments", "crawled_at", "published_at") VALUES($1, $2, $3, $4, $5, $6) 
```

```
url, source, title, comments_link, new Date()
```

The insert doesn't appear to be even

Comment (frankcash) [reply routes/lobsters.js]: Please make sure all the inserts are even

Comment (rajat19) [reply routes/lobsters.js]: Ok i will fix this

**Comment by rajat19** [comment]
Please check now

**Comment by frankcash** [comment]
Stale, please update if you wish to merge.  Would love to help if you want it.

**Pull request closed without merging**